pyseir/models/suppression_policies.py

```diff
@@ -22,6 +22,31 @@
 }
 
 
+def get_future_supression_from_r0(R0, scenario):
+    """
+    Returns the future suppression level for a given R0 and a "future scenario".
+
+    Parameters
+    ----------
+    R0:float
+        Reproduction number
+    scenario: str
+        'no_intervention', 'flatten_the_curve', 'social_distancing'.
```

Comment:
Minor: If you have an enum for this, you could use it here

Reply:
Yeah. Expected to change, so never bothered, but we should once we decide on reopening nomenclature.